components/StyledLink.js

```diff
@@ -16,7 +16,7 @@ const StyledLink = styled.a.attrs(props => {
   if (props.openInNewTab) {
     return {
       target: '_blank',
-      rel: 'noopener noreferrer',
+      rel: 'noopener noreferrer nofollow',
```

Comment:
I'm completely ignorant in SEO, but couldn't this be bad for our platform score? Meaning that the robot wouldn't be able to track internal page linking?
I think we'll need to add `nofollow` only to external links: profile website links and to the SVG widgets we allow collectives to embed in their profiles.

Reply:
I hesitated about adding this attribute when I implemented `openInNewTab`, and ended up not doing it for the following reasons:
- Some orgs (like Triplebyte) invest in OC to get some visibility among developer (ie. to hire them) and may be interested in monitoring the traffic they get back from us
- People who want this privacy usually/should have some browsers extensions that change or remove the referrer header for them on all websites
- I personally don't see that as an important privacy issue

Reply:
It's not about privacy, it's more about the idea that external websites get a boosted SEO score and do we want to be boosting the SEO visibility of certain kinds of websites. `nofollow` is saying, "Hey you can go to this external website but we aren't responsible for what's on there or endorsing it" which honestly I think isn't a bad shout. (https://webmasters.googleblog.com/2019/09/evolving-nofollow-new-ways-to-identify.html)

I've spoken to one or two people last night about this and it was suggested that the new best practice is using the `sponsored` tag. But I'm not sure if it technically applies super neatly to OC since it is meant for ads/bought content. But something to possibly consider.

(Side note, another thing to could potentially also use the `ugc` tag for any links added through comments or other future user-generated content.)

re: Triplebyte, they can monitor the traffic they get from us and where it comes from without us adding to their page ranking? Surely the 'visibility' we're offering with OC is in being present on the platform and being seen to be investing in the community, rather than pure SEO boosts? I would be a bit uncomfortable with us saying, we don't want to take away the SEO boost to these nice orgs, therefore we'll also possibly be boosting other orgs.

I agree with @kewitz that this could be further refined to only `nofollow` actual external links and I'm happy to put that work in if we want to invest in this change.

Reply:
@sbinlondon @Betree I think the quick win is to add `nofollow` to `openInNewTab`.
Reason being that this should allow scanners to navigate within OpenCollective but not jump to collective associated websites. I don't think this would impact Tripebyte's organic traffic.

Reply:
> It's not about privacy, it's more about the idea that external websites get a boosted SEO score and do we want to be boosting the SEO visibility of certain kinds of websites. nofollow is saying, "Hey you can go to this external website but we aren't responsible for what's on there or endorsing it"

Indeed! I had a misunderstanding on the purpose of this attribute.